examples/host_functions/README.md

```diff
@@ -0,0 +1,20 @@
+# Examples
+
+The `parse-json` example shows how to use a host function with a shared library `json-c`. The `parseJson` function accepts a string as a key, parses the JSON file `test.json` and returns the corresponding value.
+
+The `parse-json.wat` is a WebAssembly script. It is compiled into WebAssembly in the form of `parse-json.wasm` using the [WABT tool](https://github.com/WebAssembly/wabt).
+
+## Installation
```

Comment:
Could you please also mention your environment? The `libjson-c-dev` on different OS may have different version. It's better to have the detailed version information.

Reply:
Added: The version of `json-c` installed is `0.13.1` in `Ubuntu 20.04.3`. Is that correct?